is_core/utils/field_api.py

```diff
@@ -95,9 +96,12 @@ def get_widget(self):
             return ManyToManyReadonlyWidget
         elif field.auto_created and field.one_to_one:
```

Comment:
is this condition even required now that it is the default fallback widget? or do you keep it for clarity?

Reply:
Yes it is required. It returns ModelObjectReadonlyWidget this is reverse relation to one_to_one field

Reply:
I think @Formulka was pointing out the fact, that if you remove this condition, the behavior will be the same, thanks to newly introduced `else` logic.

Reply:
Hmm there is bug. The new else logic should return ReadonlyWidget thanks